libraries/tools/idl2k/build.gradle

```diff
@@ -0,0 +1,41 @@
+buildscript {
+    ext.antlr4_version = '4.7.1'
+}
+
+apply plugin: "antlr"
+apply plugin: "kotlin"
+
+configurePublishing(project)
```

Comment:
Please preserve the old behavior: the artifact was published only when `idl2k.deploy.skip` property is `false`

Reply:
Note that the artifact name is changed to `kotlin-idl2k` (by default it's the name of the project). It may be ok or may be not, please ask @alexander-gorshenev, are they ready for that?

Reply:
We're fine with that, I suppose.

Reply:
I've checked that the property was set only for the maven build, so we should setup something like that to preserve the same for gradle build — I can do that. 
But we do not longer need to preserve the inverted logic, i.e. publishing when the property is false, so I propose to introduce new property `idl2k.deploy` and publish only when it is set to `true`.

Reply:
@ilya-g  I agree, inverted boolean logic can be confusing

Reply:
No, by default it's `false`